src/components/lyne-teaser/styles/lyne-teaser.component.scss

```
@@ -0,0 +1,94 @@
+@mixin image-aspect-ratio-4-3 {
+  aspect-ratio: 4 / 3;
+}
+
+@mixin teaser-animation {
+  @include shadow-level-9-hard;
+
+  transform: scale(var(--scale-102));
+  transition: var(--animation-duration-4x) var(--animation-easing);
+}
+
+:host {
+  --scale-102: 1.02;
+}
+
+/* ************************************************* */
+/* Layout & spacing                                  */
+/* ************************************************* */
+.teaser {
+  border: none;
+  text-decoration: none;
+  text-align: start;
+
+  &:hover {
+    .teaser__image {
+      @include teaser-animation();
+    }
+  }
+
+  &:focus-visible {
+    @include teaser-animation();
+  }
+}
+
+.teaser__image {
+  @include image-aspect-ratio-4-3;
+  object-fit: cover;
+  border-radius: pxToRem(var(--border-radius-4x));
+  margin-right: pxToRem(var(--spacing-fixed-4x));
+  margin-bottom: 0;
+  width: 100%;
+}
+
+.teaser__inner { 
+  display: flex;
+  flex-wrap: nowrap;
+  align-items: center;
+}
+
+// Layout switch
+
+.teaser__inner {
+  .teaser--is-stacked & {
+    flex-direction: column;
+    align-items: baseline;
+  }
+}
+
+.teaser__text {
+  .teaser--is-stacked & {
+    margin-left: 0;
+  }
+}
+
+.teaser__image {
+  .teaser--is-stacked & {
+    margin-right: 0;
+    margin-bottom: pxToRem(var(--spacing-fixed-3x));
```

I don't know about your rtl requirements, but you could use

```suggestion
    margin-block-end: pxToRem(var(--spacing-fixed-3x));
```